pkg/gameserversets/controller.go

```diff
@@ -441,7 +441,7 @@ func computeReconciliationAction(strategy apis.SchedulingStrategy, list []*agone
 			handleGameServerUp(gs)
 		case agonesv1.GameServerStateReady:
 			handleGameServerUp(gs)
-		case agonesv1.GameServerStateReserved:
+		case agonesv1.GameServerStateReserved: // TODO: Isn't this already handled above in pass 1?
```

Comment:
Isn't this already handled above in pass 1? (On line 409)

Reply:
Is a good question, but this a piece of code I'm not touching 😄

Reply:
Fine by me